config/initializers/isbm_rest_adapter.rb

```diff
@@ -0,0 +1,10 @@
+Typhoeus::Config.verbose = true
+IsbmRestAdaptor.configure do |conf|
+  conf.host = 'isbm.lab.oiiecosystem.net'
+  conf.scheme = 'https'
+  conf.base_path = '/rest'
+  # conf.debugging = true
+  conf.ssl_ca_cert = "C:/Program Files/curl/curl-ca-bundle.crt"
```

Comment:
Absolute paths will cause a problem. If you need to, use an environment variable or config file for the path instead.